core/src/test/scala/org/apache/spark/serializer/KryoSerializerSuite.scala

```diff
@@ -23,8 +23,10 @@ import scala.reflect.ClassTag
 import com.esotericsoftware.kryo.Kryo
 import org.scalatest.FunSuite
 
-import org.apache.spark.{SparkConf, SharedSparkContext}
+import org.apache.spark.scheduler.HighlyCompressedMapStatus
 import org.apache.spark.serializer.KryoTest._
+import org.apache.spark.storage.BlockManagerId
+import org.apache.spark.{SharedSparkContext, SparkConf}
```

Comment:
ah, I see.  after getting it wrong elsewhere I tried relying on IntelliJ to do the ordering for me, but it does it this way.  Does [Aaron's plugin](https://plugins.jetbrains.com/plugin/7350?pr=idea) handle this correctly?  otherwise I'm gonna really need to work hard to get in this habit :P